pobax_playground/SensorialState[] points
